openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/SystemRepository.java

```diff
@@ -95,9 +99,27 @@ public Response patchSetting(String settingName, JsonPatch patch) {
 
   public void updateSetting(Settings setting) {
     try {
+      if (setting.getConfigType() == SettingsType.EMAIL_CONFIGURATION) {
+        SmtpSettings emailConfig = JsonUtils.convertValue(setting.getConfigValue(), SmtpSettings.class);
+        if (!Fernet.isTokenized(emailConfig.getPassword())) {
```

Comment:
we should just return the masked password incase if the Fernet is not configured

Reply:
done